lib/cmds/genomics_cmds/create-structural-variant-set.js

```diff
@@ -63,8 +64,7 @@ exports.handler = async argv => {
     referenceSetId: argv.reference,
     sequenceType: argv.sequenceType,
     testType: argv.testType,
-    indexedDate: argv.indexedDate,
-    performerId: argv.performerId,
```

Comment:
Why drop `performerId`?

Reply:
Dang, I completely forgot this was even out here. That's a typo, I'll get it fixed and updated and get this merged.  Since I was waiting so long for RI to say the breaking change is okay, it fell off my radar.